packages/keyring-api/CHANGELOG.md

```diff
@@ -7,6 +7,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [10.0.0]
+
+### Uncategorized
```

Comment:
We should have `Uncategorized` in the changelog (we need to check why the CI didn't fail).